app/models/aliases/ems_middleware.rb

```diff
@@ -0,0 +1 @@
+::EmsMiddleware = ::ManageIQ::Providers::MiddlewareManager
```

Comment:
@matthewd I can't remember if you wanted to keep these consistent as new providers were added, or try to avoid them.  I'll defer to you on this one.